src/scm/github.js

```diff
@@ -266,20 +287,41 @@ class Github {
     });
   }
 
+  getNamespaces() {
+    return getAllItems(Promise.resolve(
+      {
+        token: this.accessToken,
+        items: [], 
+        url: `${this.baseUrl}/user/orgs?access_token=${this.accessToken}`
+      }),
+      this.followPaginate,
+      'github'
+    )
+    .then(orgs => {
+      this.namespaces = [this.user].concat(orgs.map(org => org.login));
+      return this.namespaces;
+    })
+    .catch((err) => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/b1257aeff34c7da1703f8a8cb7e32838448492d8/src/scm/github.js#L304"></span>‘err’ is defined but never used.</td>
    </tr>
  </tbody>
</table>